content/en/tracing/services/inferred_services.md

```diff
@@ -24,9 +24,15 @@ To see inferred services, you must enable some configurations.
 Starting from version [7.60.0][1] of the Datadog Agent, these configurations are enabled by default. 
 
 {{< tabs >}}
-{{% tab "Agent v7.55.1+" %}}
+{{% tab "Agent v7.60.0+" %}}
+Starting from Datadog Agent version [7.60.0][1], no manual configuration is needed to see inferred services. The required configurations—`apm_config.compute_stats_by_span_kind` and `apm_config.peer_tags_aggregation`—are enabled by default.
 
-For Datadog Agent versions [7.55.1][2] or later, add the following to your `datadog.yaml` configuration file:
+[1]: https://github.com/DataDog/datadog-agent/releases/tag/7.60.0
+
+{{% /tab %}}
+{{% tab "Agent v7.55.1 - v7.59.1" %}}
+
+For Datadog Agent versions [7.55.1][1] through [7.59.1][2], add the following to your `datadog.yaml` configuration file:
```

Comment:
I've checked that v7.59.1 is the latest release of v7.59.x in https://github.com/DataDog/datadog-agent/releases?expanded=false&page=2